### ISSUE TITLE
Publish branch should show remote picker

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode-pull-request-github#5120]
Testing #5100

Clicking on publish here publishes to `origin`
![image](https://github.com/microsoft/vscode-pull-request-github/assets/3840081/75d1e492-4106-47ad-80f6-7f6da7c1d12d)

Should show this picker:
![image](https://github.com/microsoft/vscode-pull-request-github/assets/3840081/6333b1e7-ec2a-4963-a06e-d5cddde45ce7)

**Comment by alexr00** [comment]
I disagree. I think it should publish to whatever remote you have selected in the "Merge" picker. You have already given all the information needed to know where to publish the branch to, no need to make you say it again.

**Comment by karthiknadig** [comment]
This https://github.com/microsoft/vscode-pull-request-github/issues/5118 makes it work for me. So no need for this.
